compatibility_lib/compatibility_lib/fake_compatibility_store.py

```diff
@@ -115,10 +115,15 @@ def get_pairwise_compatibility_for_package(self, package_name: str) -> \
                frozenset([p1, p4]): [CompatibilityResult...],
             }.
         """
-        package_pairs = [[package.Package(package_name), package.Package(name)]
-                         for name in configs.PKG_LIST]
-        results = {frozenset(pair): self.get_pair_compatibility(pair)
-                   for pair in package_pairs}
+        package_pairs = [frozenset([package.Package(package_name), package.Package(name)])
+                         for name in configs.PKG_LIST
+                         if package_name != name]
+        print('package_pairs:', package_pairs)
```

Comment:
remove print statement

Reply:
I wasn't ready for review but fixed the prints and lint ;-)